using-babelfish/getting-started.md

```diff
@@ -49,13 +56,13 @@ The only mandatory flag is [`--privileged`](https://docs.docker.com/engine/refer
 
 Exposing the port \(`9432`\) with `-p 9432:9432` makes it easier to connect to the gRPC server from outside the container.
 
-Also, the path `/var/lib/bblfshd` should be mounted in the volume in order to keep bblfshd instances stateful between reboots.
+Also, if using a directory as persistent storage, the path `/var/lib/bblfshd` should be mounted in the volume in order to keep bblfshd instances stateful between reboots.
 
 If you are behind an HTTP or HTTPS proxy server, for example in corporate settings, you will need to add the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables in the docker run command to configure HTTP or HTTPS proxy behavior.
 
 ```bash
 $ docker run -d --name bblfshd --privileged -p 9432:9432 -e 
-HTTP_PROXY="http://proxy.example.com:80/" -v /var/lib/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
+HTTP_PROXY="http://proxy.example.com:80/" -v bblfshd-cache:/var/lib/bblfshd
```

Comment:
This command is missing an image name at the end

Reply:
Fixed!